drivers/usb/device/usb_dc_stm32.c

```diff
@@ -243,10 +252,105 @@ int usb_dc_set_address(const u8_t addr)
 	return 0;
 }
 
-int usb_dc_ep_start_read(u8_t ep, u8_t *data, u32_t max_data_len)
+static int usb_dc_ep_transfer(const u8_t ep, u8_t *buf, size_t dlen,
+			      bool is_in, usb_dc_transfer_callback cb)
```

Comment:
Please align.

Reply:
seems to be a github formatting issue here